modules/govuk/templates/asset_manager_extra_nginx_config.conf.erb

```diff
@@ -72,7 +72,7 @@ location ~ /cloud-storage-proxy/(.*) {
 
   # Add Google DNS server to avoid "no resolver defined to resolve"
   # errors when trying to connect to S3
-  resolver 8.8.8.8;
+  resolver 1.1.1.1;
```

Comment:
[This directive](http://nginx.org/en/docs/http/ngx_http_core_module.html#resolver) supports multiple resolvers.